packages/dart/lib/src/utils/parse_live_list.dart

```diff
@@ -539,6 +540,10 @@ class ParseLiveElement<T extends ParseObject> extends ParseLiveListElement<T> {
               super.object = parseResponse.result.first;
             }
             break;
+          case LiveQueryClientEvent.disconnected:
+            break;
+          case LiveQueryClientEvent.userDisconnected:
+            break;
```

Comment:
Just wondering, these added cases do not look like mere refactors. What is that change? If it adds or changes a feature, then it probably needs its own PR.

Reply:
If we do not consider these `case's in the new version of lint, lint will consider them as warnings and we will face problems in CI.

Reply:
Got it